function7.js

```diff
@@ -1 +1,9 @@
 // Create function calculating the take home pay. It will receive salary before tax, tax in percentage, NI in percentage.
+
+const salaryCalculator = (salary, tax, NI) => {
```

Comment:
Good, only issue here is that params should not be written in SCREAMING_SNAKE_CASE